src/utils/validation-schemas.ts

```diff
@@ -128,23 +138,23 @@ export const questSlugSchema = z
   .max(100, 'Quest slug is too long');
 
 /**
- * Schema for transaction hashes (supports both Ethereum and Solana formats)
+ * Schema for transaction hashes (supports Ethereum, Solana, UTXO, and SUI formats)
  */
 export const transactionHashSchema = z
   .string()
   .transform((val) => val) // Remove toLowerCase() as Solana signatures are case-sensitive
   .refine(
     (val) => {
-      // Check for Ethereum format (0x + 64 hex chars)
-      if (val.startsWith('0x')) {
-        return /^0x[a-f0-9]{64}$/.test(val.toLowerCase());
-      }
-      // Check for Solana format (base58)
-      return /^[1-9A-HJ-NP-Za-km-z]{88}$/.test(val); // Exact length for Solana
+      return (
+        isValidEthereumTransaction(val) ||
+        isValidSolanaTransaction(val) ||
+        isValidUTXOTransaction(val) ||
+        isValidSuiTransaction(val)
+      );
```

Comment:
I would have created a separate function for eg.
```
export const isValidTransaction = (txHash: string) => 
      isValidEthereumTransaction(txHash) ||
      isValidSolanaTransaction(txHash) ||
      isValidUTXOTransaction(txHash) ||
      isValidSuiTransaction(txHash)
```
and then call it here `isValidTransaction(val)`
as it will make things more easy to update in the future in case we need to add an extra check